$NetBSD: patch-ab,v 1.4 2010/12/05 17:13:09 gls Exp $

Something in the build fails to detect that we have wchar.h.  This
change make it build on NetBSD; a better fix is welcome.

--- src/libtom/tomcrypt_pk.h.orig	2010-08-27 04:41:55.000000000 +0000
+++ src/libtom/tomcrypt_pk.h
@@ -495,12 +495,7 @@ int der_length_printable_string(const un
 int der_printable_char_encode(int c);
 int der_printable_value_decode(int v);
 
-/* UTF-8 */
-#if (defined(SIZE_MAX) || __STDC_VERSION__ >= 199901L || defined(WCHAR_MAX) || defined(_WCHAR_T) || defined(_WCHAR_T_DEFINED)) && !defined(LTC_NO_WCHAR)
 #include <wchar.h>
-#else
-typedef ulong32 wchar_t;
-#endif
 
 int der_encode_utf8_string(const wchar_t *in,  unsigned long inlen,
                            unsigned char *out, unsigned long *outlen);
